java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -255,6 +265,8 @@ public static Object[] transactionTestResult() {
             new String[] {"one", "two"}, // zdiff(new String[] {zSetKey2, key8})
             Map.of("one", 1.0, "two", 2.0), // zdiffWithScores(new String[] {zSetKey2, key8})
             new Object[] {zSetKey2, "two", 2.0}, // bzpopmax(new String[] { zsetKey2 }, .1)
+            2L, // geoadd(geoKey1, Map.of("Palermo", new GeospatialData(13.361389, 38.115556), "Catania",
+            // new GeospatialData(15.087269, 37.502669)))
```

Comment:
You can keep this short
```suggestion
            2L, // geoadd(geoKey1, Map.of("Palermo", ..., "Catania", ...))
```